src/client/LeftMenu.jsx

```diff
@@ -1,52 +1,54 @@
-import React from "react";
-import { Link } from "react-router-dom";
-import { FaMoon, FaComments, FaHome } from "react-icons/fa";
-import { MdLeaderboard } from "react-icons/md";
+import { Link } from 'react-router-dom';
+import { FaMoon, FaComments, FaHome } from 'react-icons/fa';
+import { MdLeaderboard } from 'react-icons/md';
 
 const LeftMenu = (props) => {
-  // const {darkMode, setDarkMode} = props;
+    // const {darkMode, setDarkMode} = props;
 
-  const changeToDark = () => {
-    setDarkMode(true);
-  }
-  const changeToLight = () => {
-    setDarkMode(false);
-  }
+    const changeToDark = () => {
+        setDarkMode(true);
+    };
+    const changeToLight = () => {
+        setDarkMode(false);
+    };
```

Comment:
You could turn this into a single switch by changing `darkMode` to the opposite of what it currently is:

```js
const switchDarkMode = () => setDarkMode(!darkMode);
```